src/components/AlbumCover.jsx

```diff
@@ -0,0 +1,39 @@
+// src/components/AlbumCover.jsx
+
+import React, { useState } from 'react';
+// Importing our icon SVGs
+import heartIcon from '../assets/icons/heart.svg';
+import playIcon from '../assets/icons/play.svg';
+import dotsIcon from '../assets/icons/dots.svg';
+
+export const AlbumCover = ({ album }) => {
+    
+     // This line creates a state to track whether the album is being hovered
+     const [isHovered, setIsHovered] = useState(false);
```

Comment:
This seems to be something that you've figured out on your own. This week we'll learn more about state. But a hover state isnt really something that you should handle with a React state. Better to do that in css.